www/templates/diary/infinite_scroll_list.html

```diff
@@ -2,12 +2,15 @@
     <ion-nav-bar class="bar-stable">
     </ion-nav-bar>
     <ion-nav-buttons side="left" id="toget" class="row buttons">
-        <button ng-repeat="selF in filterInputs" ng-click="select(selF)" class="col col-50 button labelfilter" ng-class="{on:selF.state}" style="text-align: center;font-size: 14px;font-weight: 600;" translate>
-            {{selF.text}}
-        </button>
-        <button ng-click="resetSelection()" class="col col-50 button labelfilter last" ng-class="{on:allTrips}" style="text-align: center;font-size: 14px;font-weight: 600;" translate>
-            {{'.show-all'}}
-        </button>
```

Comment:
The advantage of using entries such as `col-50` is that they (at least theoretically) auto-resize on different size screens. Can't you use a lower level of `col` - e.g. `col-33` etc instead of hardcoding values?

Reply:
There are lots of options!
https://ionicframework.com/docs/v1/components/#grid-explicit

Reply:
My intent wasn't to hardcode values, it was to have the buttons automatically resize to fit the text within them, which they do now. Is that objectionable?

Reply:
Before I first got to this code, there was a `width` parameter stored in each `infinite_scroll_filter`. I changed it to use the `col` values because that _was_ hardcoded (and not accommodating of multiple languages), but my new solution seems even less hardcoded.